Flow.Launcher.Test/FilesFoldersTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Flow.Launcher.Plugin.SharedCommands;
+using NUnit.Framework;
+
+namespace Flow.Launcher.Test
+{
+    [TestFixture]
+    
+    public class FilesFoldersTest
+    {
+        // Testcases from https://stackoverflow.com/a/31941905/20703207
+        // Disk
+        [TestCase(@"c:", @"c:\foo", true)]
```

Comment:
This failed the test. Using `c:` works in Win Explorer but not supported in Explorer plugin, is this test still applicable?

Reply:
It passes my local test. Weird. Should I disable it for CI?